Framework/src/CcdbDatabase.cxx

```diff
@@ -198,4 +249,18 @@ void CcdbDatabase::truncate(std::string taskName, std::string objectName)
   ccdbApi.truncate(taskName + "/" + objectName);
 }
 
+void CcdbDatabase::storeStreamerInfosToFile(std::string filename)
+{
+  TH1F* h1 = new TH1F("asdf", "asdf", 100, 0, 99);
+  shared_ptr<MonitorObject> mo1 = make_shared<MonitorObject>(h1, "fake");
+  TMessage message(kMESS_OBJECT);
+  message.Reset();
+  message.EnableSchemaEvolution(true);
+  message.WriteObjectAny(mo1.get(), mo1->IsA());
+  TList* infos = message.GetStreamerInfos();
```

Comment:
I am not sure if understand it. Does a message with any ROOT object contain the streaming information for all of the other objects, even the custom ones?

Reply:
Yes. ROOT loads the streamer info for the object that is written in the TMessage (because we called EnableSchemaEvolution), including custom objects. It loads the streamer info for the whole hierarchy of classes and for all members of the class.